src/main/java/com/iotiq/application/domain/MaterialParameter.java

```diff
@@ -0,0 +1,21 @@
+package com.iotiq.application.domain;
+
+import jakarta.persistence.Embeddable;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.math.BigDecimal;
+
+@Getter
+@Setter
+@Embeddable
+@AllArgsConstructor
+@NoArgsConstructor
+public class MaterialParameter {
+    private BigDecimal thickness;
```

Comment:
Why is this a BigDecimal? Does it require that much precision?

Reply:
i used float istead of bigdecimal